src/molecules/link-card/link-card.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { LinkCardTypes } from "../constants/link-card-types";
+import { StringUtils } from "andculturecode-javascript-core";
+import { Anchor } from "../../atoms/anchors/anchor";
+import { Button } from "../../atoms/buttons/button";
+import { Icon } from "../../atoms/icons/icon";
+import { Icons } from "../../atoms/constants/icons";
+import { Paragraph } from "../../atoms/typography/paragraph";
+import { ParagraphSizes } from "../../atoms/constants/paragraph-sizes";
+
+// -------------------------------------------------------------------------------------------------
+// #region Contants
+// -------------------------------------------------------------------------------------------------
+
+const COMPONENT_CLASS = "c-link-card";
+
+// #endregion Constants
+
+// -------------------------------------------------------------------------------------------------
+// #region Interfaces
+// -------------------------------------------------------------------------------------------------
+
+export interface LinkCardProps {
+    children: any;
+    includeIcon?: boolean;
+    label: string;
+    to?: any;
+    type?: LinkCardTypes;
+    onClick?: () => void;
+}
+
+// #endregion Interfaces
+
+// -------------------------------------------------------------------------------------------------
+// #region Component
+// -------------------------------------------------------------------------------------------------
+
+const LinkCard: React.FC<LinkCardProps> = (props: LinkCardProps) => {
+    const cssClassNames = [COMPONENT_CLASS];
+    if (props.includeIcon) {
+        cssClassNames.push("-with-icon");
+    }
+    const cssClassNamesFlat = cssClassNames.join(" ");
+
+    const type = props.type ?? LinkCardTypes.Link;
+
+    const renderChildren = () => (
+        <React.Fragment>
+            {// if
+            props.includeIcon && <Icon type={Icons.Lightbulb} />}
```

Comment:
should this icon be configurable via a prop instead of a boolean?

Reply:
I'll make the type optionally configured and defaulted to `LightBulb`